lib/moonshot/build_mechanism/github_release.rb

```diff
@@ -140,6 +143,20 @@ def hub_release_exists(semver)
       false
     end
 
+    def hub_find_remote_tag(tag_name)
+      attempts = 0
+      # Attempt to find the build. Rescue and re-attempt if the build can not
+      # be found on github yet.
+      begin
+        sleep 1 # Give some time for tag to showup.
```

Comment:
Yeah, we can do that. I put it here because the first attempt is definitely bound to fail. We can accept that cos it will retry or not remove sleep from line 116